PrivateCollections/Germany/Neukirchen005.xml

```diff
@@ -196,24 +309,45 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </objectDesc>
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
-                        <seg type="script"/>
+                        <seg type="script">Mid-16th to mid-17th century.</seg>
```

Comment:
take out this is not seg but date

Reply:
What exactly should be taken out?
Cp. guidelines:
handNote script="Ethiopic" xml:id="h1" corresp="#p1.2 #p1.3 #a4"
            seg type="script"> 15th Century square script?</seg>
            seg type="ink">Black, red</seg>
            ...
        </handNote>

Reply:
ok, i normally mark pure dates as `<date>`, and as seg type script only if there is some additional information that follows, but it is OK if the guidelines say so